Event Signup/event-signup.js

```diff
@@ -0,0 +1,32 @@
+document.getElementById('event-sign-up-form').addEventListener('submit',function(event){
+    // Prevent default submission
+    event.preventDefault();
+    // Collect form data.
+    const eventName = document.getElementById('event-name').value;
+    const repName = document.getElementById('rep-name').value;
+    const repEMail= document.getElementById('rep-email').value;
+    const role = document.getElementById('role').value;
+
+    //Validate inputs
+    if(!eventName || !repName || !repEMail || !role){
+        alert('Pleae fill out all required fields.')
```

Comment:
does this still work if the input is "        "?

Reply:
otherwise, this is a nice and concise way of determining if an input is filled in